packages/tocco-theme/src/ToccoTheme/theme.scss

```diff
@@ -0,0 +1,32 @@
+/* tocco theme for react components
+ * CSS class .tocco-ui-theme is used to scope styles.
+ * CSS class .tocco-ui-theme is used to increase specificity compared to standard vendors implementation. Hence Tocco's Bootstrap has a higher specificity than Bootstrap.
```

Comment:
A Stylelint rule does exist to limit amount of characters (https://stylelint.io/user-guide/rules/max-line-length/). I prefer not to limit it but to configure my editor for good readability.